README.md

```diff
@@ -13,6 +13,16 @@
     └── openshift               # Build and Deploy Configurations.
     └── bin                     # Useful scripts.
 
+## RUN
+
+Create a `.env` file from the `.env.template`
+
+Confiture the necessary variables in `.env`
```

Comment:
type "configure'

Reply:
lol confiture means jam in french

Reply:
good to know. It is up to you to change it or not.